packages/cli-e2e/__tests__/vue.specs.ts

```diff
@@ -52,22 +97,37 @@ describe('ui:create:vue', () => {
     await buildTerminalExitPromise;
   };
 
-  const startApplication = async (processManager: ProcessManager) => {
+  const startApplication = async (
+    processManager: ProcessManager,
+    debugName = 'vue-server',
+    errorCallback: (error: string) => void = () => {}
+  ) => {
     const serverTerminal = new Terminal(
       'npm',
       ['run', 'start'],
       {
         cwd: getProjectPath(projectName),
       },
       processManager,
-      'vue-server'
+      debugName
```

Comment:
Otherwise we have multiple `startApplication` Terminals writing in the same stderr and stdout files